dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -0,0 +1,38 @@
+class LearningGoalEvaluationsController < ApplicationController
+  authorize_resource
+
+  def create
+    learning_goal_evaluation = LearningGoalEvaluation.new(learning_goal_evaluation_params)
+    learning_goal_evaluation.teacher_id = current_user.id
+
+    learning_goal_evaluation.save!
+
+    render json: learning_goal_evaluation
+  end
+
+  def update
+    params.require(:id)
+    learning_goal_evaluation = LearningGoalEvaluation.find_by_id(params[:id])
+
+    if learning_goal_evaluation.teacher_id == current_user.id
+      learning_goal_evaluation.update(learning_goal_evaluation_params)
+      render json: learning_goal_evaluation
```

Comment:
This code will fail silently If the `update` doesn't validate or otherwise fails. Try this instead:

```suggestion
      if learning_goal_evaluation.update(learning_goal_evaluation_params)
        render json: learning_goal_evaluation
      else
        head :bad_request
```

`update!` would also be ok. That raises an exception if the update fails, which you could then handle.